test-cases/tests/dashboards/e04-verify-slo-dashboard.md

```diff
@@ -31,6 +31,10 @@ Note: this test should only be performed at a time it will not affect other ongo
    > redhat-rhmi-solution-explorer -> Workloads -> Deployment Configs -> Scale to 1
    > redhat-rhmi-ups -> Workloads -> Deployments -> Scale to 1
 
+Execute the scripts below in a separate terminal tab/window and keep them running. It will keep scaling down stuff so that things don't start working automatically.
```

Comment:
```suggestion
Execute the scripts below in a separate terminal tab/window and keep them running. It will keep scaling down resources and prevent from automatic scale up by the operator(s).
```

Reply:
hey, resources is much nicer word than stuff :) Was not sure how to call Deployments, DeploymentConfigs, and StatefulSets by general word.